### PR TITLE
07-add-calc

### PR DESCRIPTION
**Pull request by Albert-SA**

### CODE REVIEW

File: src/ts/modules/calc.ts [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+    const optionBlock: any = document.querySelector(option);
+    const promocodeBlock: any = document.querySelector(promocode);
+    const priceBlock: any = document.querySelector(price);
+
+    let sum: number = 0;
+    const calcSum = () => {
+        if (!sizeBlock && !materialBlock && !optionBlock) return;
+        sum = Math.round((+sizeBlock.value) * (+materialBlock.value) + (+optionBlock.value));
+
+        if ((sizeBlock.value == '') || (materialBlock.value == '')) {
```
Comment (andrey-kudinov) [inline src/ts/modules/calc.ts]: `if (!sizeBlock.value || !materialBlock.value) { ...`

File: src/ts/modules/calc.ts [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+            priceBlock.textContent = 'ВВЕДИТЕ НЕДОСТАЮЩИЙ РАЗМЕР ЛИБО МАТЕРИАЛ';
+        } else if (promocodeBlock.value === 'IWANTPOPART') {
```
Comment (andrey-kudinov) [inline src/ts/modules/calc.ts]: `promocodeBlock.value.toUpperCase()`

**Review by andrey-kudinov — CHANGES_REQUESTED** [review]

File: src/ts/modules/calc.ts (hunk 1 above)
Comment (Albert-SA) [reply src/ts/modules/calc.ts]: ok

File: src/ts/modules/calc.ts (hunk 2 above)
Comment (Albert-SA) [reply src/ts/modules/calc.ts]: ok

**Pull request merged**